core/store/models/log_events.go

```diff
@@ -85,21 +85,33 @@ func TopicFiltersForRunLog(logTopics []common.Hash, jobID *ID) [][]common.Hash {
 }
 
 // FilterQueryFactory returns the ethereum FilterQuery for this initiator.
-func FilterQueryFactory(i Initiator, from *big.Int) (ethereum.FilterQuery, error) {
+func FilterQueryFactory(i Initiator, nextHead *big.Int) (ethereum.FilterQuery, error) {
```

Comment:
What was the reason for the rename?  I feel 'from' was better than 'nextHead', nextHead reads as an attribute for iteration as opposed to a filter.  What about 'fromBlock', as the same from the filter query?

Reply:
I considered `fromBlock`, but the InitiatorParams already have a `fromBlock` field.  It wouldn't be entirely clear why two fromBlocks would be passed in and one would be ignored sometimes. 
 Consequently, I felt it was semantically cleaner and more straightforward to simply specify that it's the next head.

Reply:
Thanks for the explanation, though unfortunately I disagree.  Would anyone else like to give an opinion?